<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="tracelayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>&lt;status&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="traceView">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="queueFrame">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <item row="1" column="0" colspan="3">
         <widget class="QListWidget" name="queueView"/>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>118</width>
            <height>26</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="clearQueue">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="deleteQueueItem">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Queued commands:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionBreak">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/break</normaloff>:/break</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause the script</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/continue</normaloff>:/continue</iconset>
   </property>
   <property name="text">
    <string>&amp;Continue</string>
   </property>
   <property name="toolTip">
    <string>Continue script</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/stop</normaloff>:/stop</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop script (in scan stop after current point)</string>
   </property>
  </action>
  <action name="actionEmergencyStop">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/emergency</normaloff>:/emergency</iconset>
   </property>
   <property name="text">
    <string>&amp;Immediate Stop</string>
   </property>
  </action>
  <action name="actionFinish">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/finish</normaloff>:/finish</iconset>
   </property>
   <property name="text">
    <string>Stop after line</string>
   </property>
   <property name="toolTip">
    <string>Finishes the current line and stops the script</string>
   </property>
  </action>
  <action name="actionBreakCount">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/pausecount</normaloff>:/pausecount</iconset>
   </property>
   <property name="text">
    <string>Pause counting</string>
   </property>
   <property name="toolTip">
    <string>Pause during count, or after current step in script</string>
   </property>
  </action>
  <action name="actionBreak2">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/break</normaloff>:/break</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause the script</string>
   </property>
  </action>
  <action name="actionStop2">
   <property name="icon">
    <iconset resource="../../../../../resources/nicos-gui.qrc">
     <normaloff>:/stop</normaloff>:/stop</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop script (in scan stop after current point)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
